.github/workflows/vercel-error-monitor.yml

```
@@ -0,0 +1,27 @@
+name: Vercel Error Monitor
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)